src/explore-education-statistics-frontend/src/modules/table-tool/components/MultiHeaderTable.tsx

```diff
@@ -0,0 +1,169 @@
+                        </th>
+                      );
+                    }),
+                  )}
+              </tr>
+            );
+          })}
+        </thead>
+
+        {!isStickyHeader && (
+          <tbody>
+            {rowHeaders
+              .reduce<ReactElement[][]>(
+                (acc, headerGroup, headerGroupIndex) =>
+                  acc.flatMap((row, rowIndex) =>
+                    headerGroup.map((header, index) => {
+                      const rowSpan = getSpan(rowHeaders, headerGroupIndex);
+                      const isRowGroup =
+                        headerGroupIndex !== rowHeaders.length - 1;
+
+                      const headerCell = (
+                        <th
+                          // eslint-disable-next-line react/no-array-index-key
+                          key={`${rowIndex}_${headerGroupIndex}_${index}`}
+                          className={classNames({
+                            'govuk-table__cell--numeric': !isRowGroup,
+                            'govuk-table__cell--center': isRowGroup,
+                            [styles.borderRight]: !isRowGroup,
+                            [styles.borderBottom]:
+                              isRowGroup || index === headerGroup.length - 1,
+                          })}
+                          rowSpan={rowSpan}
+                          role={isRowGroup ? 'rowgroup' : 'row'}
+                        >
+                          {header}
+                        </th>
+                      );
+
+                      if (index === 0) {
+                        return [...row, headerCell];
+                      }
+
+                      return [...row.slice(headerGroupIndex), headerCell];
+                    }),
+                  ),
+                [[]],
+              )
+              .map((headerCells, rowIndex) => (
+                <tr
+                  // eslint-disable-next-line react/no-array-index-key
+                  key={rowIndex}
+                  className={classNames({
+                    [styles.borderBottom]: (rowIndex + 1) % firstRowSpan === 0,
+                  })}
+                >
+                  {headerCells}
+                  {!isStickyColumn && (
+                    <>
+                      {rows[rowIndex].map((cell, cellIndex) => (
+                        <td
+                          // eslint-disable-next-line react/no-array-index-key
+                          key={cellIndex}
+                          className={classNames('govuk-table__cell--numeric', {
+                            [styles.borderLeft]: cellIndex % firstColSpan === 0,
+                          })}
+                        >
+                          {cell}
+                        </td>
+                      ))}
+                    </>
+                  )}
+                </tr>
+              ))}
+          </tbody>
+        )}
+      </table>
+    );
+  },
+);
+
+MultiHeaderTable.displayName = 'MultiHeaderTable';
```

Comment:
Why the need for this?  Is it not "MultiHeaderTable" by default?

Reply:
This is actually a lint from wrapping the `MultiHeaderTable` with the `forwardRef` function. `forwardRef` is an example of a higher-ordered component (HOC) which creates some new  component from the one we actually pass in as an argument. 

I think the consequence of this is that it doesn't automatically attach the correct display name for the returned component, so we have to attach it manually. Note, this is only really for our benefit when using React Dev Tools.